source/administration/config-settings.rst

```diff
@@ -112,6 +123,42 @@ Enable Team Directory ``"EnableTeamListing": false``
 
 ________
 
+Policy (Enterprise)
+``````````````````````````
+Settings to configure the permission restrictions for sending team invite links and managing channels.  
+
+Enable sending team invites from ``"RestrictTeamInvite": "all"``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Restrict the permission levels required to send team invites. Note: if permissions are restricted and "Get Team Invite Link" is used to share a link, it will need to be regenerated after the desired users joined the team.
```

Comment:
I'm not quite sure what this sentence means.. 
1) Invite link regenerated where? Or do you mean just obtaining a second invite link to another user?
2) Also, `if permissions are restricted` - in what way? When they're restricted to admins? Or to system admins?